appveyor.yml

```diff
@@ -7,14 +7,22 @@ environment:
   global:
     DOWNLOAD_DIR: _download_cache
     APPVEYOR_SAVE_CACHE_ON_ERROR: true
-    PATH: C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Program Files\Git\cmd
+    PATH: C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Program Files\Git\cmd;C:\Program Files\7-Zip
   matrix:
     - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
       VS_VER: "14"
       PLATFORM: "x86"
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
```

Comment:
same here 2015?

I also think we should probably build for 2015 and 2017 (can be in a different patch though)

Reply:
see previous reply.